src/Application/ApplyRulesUseCase.php

```diff
@@ -4,21 +4,36 @@
 
 namespace ProfessionalWiki\Rules\Application;
 
+use Iterator;
+use MediaWiki\Title\Title;
+use WikiPage;
+
 class ApplyRulesUseCase {
 
 	public function __construct(
 		private readonly RuleListLookup $ruleListLookup,
 	) {
 	}
 
-	public function applyToPage( /* TODO: some page identifier */ ): void {
+	public function applyToPage( WikiPage $wikiPage ): void {
```

Comment:
That's not an identifier. I was thinking TitleValue, page ID, or similar. Otherwise the maintenance script needs to create these WikiPage objects and feed them to the UseCase